reconciletags/reconciletags.py

```diff
@@ -65,15 +65,16 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--dry-run', dest="dry_run",
                         action='store_true', default=False)
-    parser.add_argument('file',
+    parser.add_argument('files',
```

Comment:
I think you want to use something like nargs='+' here to properly support a repeated argument:
https://docs.python.org/3/library/argparse.html#example